src/components/Modal/src/ModalLayer.vue

```diff
@@ -2,16 +2,16 @@
 	<div :class="$s.Layer">
 		<m-transition-fade-in>
 			<div
-				v-if="currentLayer.state.vnode"
+				v-if="currentModalLayer.state.vnode"
```

Comment:
Why the name change?

We may need to consider that more than modals can exist at this layer stack in the future and currentLayer may be more descriptive.

Reply:
I can probably put it back. There was a naming conflict with the popover layer but I ended up not using it. 

Though I'm not sure what you mean that there could be more than modals? This is specifically a modal layer.